frontend/package.json

```diff
@@ -24,7 +24,7 @@
     "typescript": "^4.4.3",
     "web-vitals": "^1.1.2"
   },
-  "proxy": "http://localhost:3553/",
+  "proxy": "http://localhost:5555/",
```

Comment:
I normally use 5555, but that started to cause issues so I went back to 3553.  You might want to keep it as 3553 so this proxy isn't always changing.  You would also have to change your .env file.